src/sonic_ax_impl/mibs/ietf/rfc4363.py

```diff
@@ -99,7 +99,6 @@ def update_data(self):
 
             vlanmac = self.fdb_vlanmac(fdb)
             if not vlanmac:
-                mibs.logger.error("SyncD 'ASIC_DB' includes invalid FDB_ENTRY '{}': failed in fdb_vlanmac().".format(fdb_str))
```

Comment:
Can we keep this as logger.debug so we don't log by default but if we have verbose log turned on we can still see it? 
BTW:  Do we know in what conditions the FDB entry could be invalid?